java/piranha/src/test/java/com/uber/piranha/XPFlagCleanerTest.java

```diff
@@ -2237,16 +2044,34 @@ public void exceptionWhenConfigFileInvalid() throws Exception {
     b.putFlag("Piranha:IsTreated", "true");
     b.putFlag("Piranha:Config", "src/test/resources/config/invalid/properties_test_invalid.json");
 
-    expectedEx.expect(ParseException.class);
-    expectedEx.expectMessage("Invalid or incorrectly formatted config file.");
+    expectedEx.expect(PiranhaConfigurationException.class);
+    expectedEx.expectMessage("methodProperties not found.");
+
+    XPFlagCleaner flagCleaner = new XPFlagCleaner();
+    flagCleaner.init(b.build());
+  }
+
+  /*
+   * Check that the user is warned when using an old style piranha.properties file
+   * */
+  @Test
+  public void warningWhenConfigFileOld() throws Exception {
```

Comment:
This test is new, everything else in this file is removing old `try{...}catch{...}` blocks and updating error messages.